pkg/scaffold/terraform.go

```diff
@@ -40,7 +42,15 @@ const outputTemplate = `output "{{ .Name }}" {
 func (scaffold *Scaffold) handleTerraform(wk *wkspace.Workspace) error {
 	repo := wk.Installation.Repository
 	providerCtx := buildContext(wk, repo.Name, wk.Terraform)
-	backend, err := wk.Provider.CreateBackend(repo.Name, providerCtx)
+
+	var providerVersions semver.ByVersion
+
+	for i := range wk.Terraform {
+		providerVersions = append(providerVersions, wk.Terraform[i].Terraform.Dependencies.ProviderVsn)
+	}
+
+	// use the latest version of the TF template for the provider
+	backend, err := wk.Provider.CreateBackend(repo.Name, providerVersions[providerVersions.Len()-1], providerCtx)
```

Comment:
I think you still need to invoke sort on this right?

Reply:
Good point, looks like I removed that accidentally.